### PR TITLE
devise installed

### PR DESCRIPTION
**Pull request by Vitaliy-Maznyi**
devise and basic views installed. I am waiting for User model to be able to setup devise

### CODE REVIEW

File: config/locales/devise.en.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
```
Comment (DenisUA) [inline config/locales/devise.en.yml:1]: Do you need this file?

Comment (Vitaliy-Maznyi) [reply config/locales/devise.en.yml:1]: Will we do i18n of our project? @DenisUA

Comment (DenisUA) [reply config/locales/devise.en.yml:1]: Then please add new locals

**Comment by yashka713** [comment]
@Vitaliy-Maznyi :+1:

**Pull request merged**